automated-orders/packages/utils-library/src/services/cache.ts

```diff
@@ -0,0 +1,52 @@
+export class LruCache<T> {
+
+    constructor(maxEntries: number) {
+        this.maxEntries = maxEntries;
+    }
+
+    private values: Map<string, T> = new Map<string, T>();
```

Comment:
are you sure that using a Map is enough?

i don't understand how this keeps track of the order of the entries, i.e. how to decide which entry to delete when maxEntries is reached

Reply:
Yes, using a map was sufficient, with the `put` method doing the trick, but I replaced it with an LRU cache from NPM.